lib/entry.js

```diff
@@ -14,14 +14,16 @@ function Entry(exports) {
   this.exports = exports;
   // The number of times this.runSetters has been called.
   this.runCount = 0;
-  // Setters for assigning to local variables in parent modules.
-  this.setters = new FastObject;
   // Getters for local variables exported from this module.
-  this.getters = new FastObject;
+  this.getters = Object.create(null);
+  // Setters for assigning to local variables in parent modules.
+  this.setters = Object.create(null);
 }
 
 var Ep = Entry.prototype;
 
+Ep.__proto__ = null;
```

Comment:
`Object.setPrototypeOf` is OK in the Node 4+ code paths just as `Object.assign`, classes, `let`, and `const` are OK. Setting the `__proto__` is more aligned with the `Object.setPrototypeOf` use since it keeps the `prototype` object intact and just mods the inheritance of the `prototype` (`Object.setPrototypeOf` is the thing that `__proto__` became). There is no compat concern here.